mavros_extras/src/plugins/odom.cpp

```diff
@@ -341,46 +232,30 @@ class OdometryPlugin : public plugin::PluginBase {
 		Matrix6d r_vel = Matrix6d::Zero();	//!< Zero initialized velocity 6-D Covariance matrix. WRT child_frame_id
 
 		mavlink::common::msg::ODOMETRY msg {};
+		msg.frame_id = utils::enum_value(MAV_FRAME::VISION_NED);
```

Comment:
We deprecated this. https://mavlink.io/en/messages/common.html#MAV_FRAME_VISION_NED

Reply:
@mhkabir How do I get the LOCAL_FRD frame? Do I have to add it manually in enum_to_string.cpp?